src/main/scala/com/kinja/amqp/AmqpClientFactory.scala

```diff
@@ -15,25 +15,23 @@ class AmqpClientFactory {
 		actorSystem: ActorSystem,
 		logger: Logger,
 		ec: ExecutionContext,
-		getWriteConnection: () => Connection,
-		getReadConnecion: () => Connection,
-		hostname: String
+		messageStores: Map[String, MessageStore]
```

Comment:
What's the String here? I know what it is, but as a user of the API who's unfamiliar with it, a String parameter doesn't tell anything. Is it the message store name maybe?

Reply:
Made it into `DeliveryGuarantee`.